### PR TITLE
show detail order

### PR DESCRIPTION
**Pull request by nguyenhonghai1997bg**

### CODE REVIEW

**Comment by nguyenhonghai1997bg** [comment]
ready

File: project1/app/Http/Controllers/Admin/DetailOrderController.php [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+use App\Order;
+use App\Detail_order;
+
+class DetailOrderController extends Controller
+{
+    public function show($id)
+    {
+        try {
+            $order = Order::findOrFail($id);
+            $detailOrders = Detail_order::where('order_id', $id)->get();
```
Comment (BunnyPi04) [inline project1/app/Http/Controllers/Admin/DetailOrderController.php]: viết vào repository nhé

**Comment by BunnyPi04** [comment]
commented

**Comment by nguyenhonghai1997bg** [comment]
ready

File: project1/routes/web.php [hunk 2]
```diff
@@ -104,7 +106,9 @@
     Route::post('statistical', 'OrderDetailController@statistical')->name('statistical');
     Route::post('get-year-order', 'OrderDetailController@getYearOrder');
     Route::post('unpublish-order', 'OrderDetailController@unpublish');
+    Route::post('processed', 'OrderDetailController@processed');
     Route::get('order-success', 'OrderDetailController@orderSuccess')->name('order-success');
+    Route::get('order-processed', 'OrderDetailController@orderProcessed')->name('order-processed');
```
Comment (BunnyPi04) [inline project1/routes/web.php]: https://viblo.asia/p/cac-convention-thuong-gap-khi-moi-bat-dau-training-php-tai-framgia-bJzKmgq6l9N#_12-dat-ten-trong-laravel-4
Tên route đặt snake_case + dot notation

**Comment by BunnyPi04** [comment]
commented

**Comment by nguyenhonghai1997bg** [comment]
ready

**Pull request merged**